### PR TITLE
18.2.4 Release notes

### PR DESCRIPTION
**Pull request by yuriw**

### CODE REVIEW

**Comment by yuriw** [comment]
https://wip-yuriw-release-18.2.3-post-main.ceph.io/en/news/blog/2024/v18-2-3-reef-released/

File: src/en/news/blog/2024/v18-2-3-reef-released/index.md [hunk 1]
```diff
@@ -0,0 +1,815 @@
...
+
+- doc/mgr: update zabbix information ([pr#56631](https://github.com/ceph/ceph/pull/56631), Zac Dover)
+
+- doc/rados/configuration/bluestore-config-ref: Fix lowcase typo ([pr#54694](https://github.com/ceph/ceph/pull/54694), Adam Kupczyk)
+
+- doc/rados/configuration/osd-config-ref: fix typo ([pr#55678](https://github.com/ceph/ceph/pull/55678), Pierre Riteau)
+
+- doc/rados/operations: add EC overhead table to erasure-code<span></span>.rst ([pr#55244](https://github.com/ceph/ceph/pull/55244), Anthony D'Atri)
+
+- doc/rados/operations: document `ceph balancer status detail` ([pr#55263](https://github.com/ceph/ceph/pull/55263), Laura Flores)
```
Comment (ljflores) [inline src/en/news/blog/2024/v18-2-3-reef-released/index.md:211]:
```suggestion
```

Remove this; it was backported.

File: src/en/news/blog/2024/v18-2-3-reef-released/index.md [hunk 2]
```diff
@@ -0,0 +1,815 @@
...
+
+- qa: use exisitng ignorelist override list for fs:mirror[-ha] ([issue#62482](http://tracker.ceph.com/issues/62482), [pr#54766](https://github.com/ceph/ceph/pull/54766), Venky Shankar)
+
+- radosgw-admin: 'zone set' won't overwrite existing default-placement ([pr#55061](https://github.com/ceph/ceph/pull/55061), Casey Bodley)
+
+- rbd-nbd: fix resize of images mapped using netlink ([pr#55316](https://github.com/ceph/ceph/pull/55316), Ramana Raja)
+
+- reef backport: rook e2e testing related PRs ([pr#55375](https://github.com/ceph/ceph/pull/55375), Redouane Kachach)
+
+- Revert "doc/rados/operations: document `ceph balancer status detail`" ([pr#55358](https://github.com/ceph/ceph/pull/55358), Laura Flores)
```
Comment (ljflores) [inline src/en/news/blog/2024/v18-2-3-reef-released/index.md:741]:
```suggestion
```
Remove this

File: src/en/news/blog/2024/v18-2-3-reef-released/index.md [hunk 3]
```diff
@@ -0,0 +1,815 @@
...
+
+- qa: `fs volume rename` requires `fs fail` and `refuse\_client\_session` set ([issue#64174](http://tracker.ceph.com/issues/64174), [pr#56171](https://github.com/ceph/ceph/pull/56171), Venky Shankar)
+
+- qa: Add benign cluster warning from ec-inconsistent-hinfo test to ignorelist ([pr#56151](https://github.com/ceph/ceph/pull/56151), Sridhar Seshasayee)
+
+- qa: add centos\_latest (9<span></span>.stream) and ubuntu\_20<span></span>.04 yamls to supported-all-distro ([pr#54677](https://github.com/ceph/ceph/pull/54677), Venky Shankar)
+
+- qa: add diff-continuous and compare-mirror-image tests to rbd and krbd suites respectively ([pr#55928](https://github.com/ceph/ceph/pull/55928), Ramana Raja)
+
+- qa: add support/qa for cephfs-shell on CentOS 9 / RHEL9 ([pr#54865](https://github.com/ceph/ceph/pull/54865), Patrick Donnelly)
```
Comment (ljflores) [inline src/en/news/blog/2024/v18-2-3-reef-released/index.md:703]:
```suggestion
```
Remove this;  it was reverted

File: src/en/news/blog/2024/v18-2-3-reef-released/index.md [hunk 4]
```diff
@@ -0,0 +1,815 @@
...
+
+- radosgw-admin: 'zone set' won't overwrite existing default-placement ([pr#55061](https://github.com/ceph/ceph/pull/55061), Casey Bodley)
+
+- rbd-nbd: fix resize of images mapped using netlink ([pr#55316](https://github.com/ceph/ceph/pull/55316), Ramana Raja)
+
+- reef backport: rook e2e testing related PRs ([pr#55375](https://github.com/ceph/ceph/pull/55375), Redouane Kachach)
+
+- Revert "doc/rados/operations: document `ceph balancer status detail`" ([pr#55358](https://github.com/ceph/ceph/pull/55358), Laura Flores)
+
+- Revert "reef: qa: add support/qa for cephfs-shell on CentOS 9 / RHEL9" ([pr#55290](https://github.com/ceph/ceph/pull/55290), Venky Shankar)
```
Comment (ljflores) [inline src/en/news/blog/2024/v18-2-3-reef-released/index.md:743]:
```suggestion
```
Remove this

**Comment by ljflores** [comment]
Any other comments on https://github.com/ceph/ceph/pull/56932/ should also be applied here.

**Review by ljflores — APPROVED** [review]

**Comment by dvanders** [comment]
@ljflores do we want to merge this?

**Review by jdurgin — APPROVED** [review]

**Pull request merged**